alf/algorithms/config.py

```diff
@@ -35,7 +35,6 @@ def __init__(self,
                  num_checkpoints=10,
                  confirm_checkpoint_upon_crash=True,
                  no_thread_env_for_conf=False,
-                 load_checkpoint_strict=True,
```

Comment:
why remove this option?

Reply:
> why remove this option?

Because now `Checkpointer.load()` can be directly configured.